dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.io.netty.feature

```diff
@@ -1,8 +0,0 @@
--include= ~${workspace}/cnf/resources/bnd/feature.props
-symbolicName=io.openliberty.io.netty
```

Comment:
We're discussing on the Netty team what we should rename these to.

Reply:
As I commented in the PR as well.  Why would you need a feature for a single bundle?  Isn't the existing internal netty feature sufficient?

Reply:
Jared, we're all set here. I approved.